src/gui/Src/BasicView/StdTable.cpp

```diff
@@ -195,6 +237,55 @@ void StdTable::expandSelectionUpTo(int to)
     }
 }
 
+void StdTable::expandUp()
+{
+    int wRowIndex = mSelection.firstSelectedIndex - 1;
+    if(wRowIndex >= 0)
+    {
+        if(wRowIndex < mSelection.fromIndex)
+        {
+            mSelection.fromIndex = wRowIndex;
+            mSelection.firstSelectedIndex = wRowIndex;
+            mSelection.toIndex =  mSelection.toIndex ;

```

Comment:
Are the self assignments in expandUp and expandDown intended? They may cause warnings with certain compiler flags or static analysis tools